inst/workflow/1_mappings/SITE.yaml

```diff
@@ -35,6 +36,8 @@ spec:
       type: character
     subjid:
       type: character
+    Status:
```

Comment:
we can remove this Status, right? looks like a duplicate

Reply:
This one captures subject status, to count the number of active subjects at the site.